### PR TITLE
upload landscape-report.md

### PR DESCRIPTION
**Pull request by aaronsnow**

### CODE REVIEW

File: landscape-report.md [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+
+
+## About this Document
+
+This report was released April 2021 under a [Creative Commons Attribution-ShareAlike 4.0 license](https://creativecommons.org/licenses/by-sa/4.0/), and should be cited as: Waldo Jaquith and Carnahan, Robin. _Sharing Government Software: How Agencies Are Cooperatively Building Mission-Critical Software_. Beeck Center for Social Impact + Innovation, Georgetown University, 2021.
+
+
+
+
+## Table of Contents
```
Comment (switzersc) [inline landscape-report.md:38]: It'd be helpful to turn these list items into links to their relevant sections (which _should_ be doable with markdown links e.g. `[Executive Summary](#Executive-Summary)`)

File: landscape-report.md [hunk 2]
```diff
@@ -0,0 +1,418 @@
+---
+title: "Sharing Government Software"
+---
```
Comment (switzersc) [inline landscape-report.md:3]: thoughts on where this should live in the site? we don't have to link there yet but want to get the convo started

File: landscape-report.md [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+	- Built Externally, Agencies Contribute
+	- Top-Down
+	- Built Open, Then Privatized
+	- Built Commercialized
+- Appendix B: Inventory of cooperatively developed software projects
+
+
+
+
+## Executive Summary
```
Comment (switzersc) [inline landscape-report.md:84]: Just realized this isn't in the table of contents

**Review by switzersc — COMMENTED** [review]
One suggestion to use links for the table of contents

**Comment by switzersc** [comment]
Oh also, what do you think about adding a link to the original report at the top or bottom of the page? Something simple like "Download the original report [here](link)."

File: landscape-report.md (hunk 2 above)
Comment (aaronsnow) [reply landscape-report.md:3]: Probably a section along the left called "Reports", with this and the other reports that are up at the Beeck site?

File: landscape-report.md (hunk 1 above)
Comment (aaronsnow) [reply landscape-report.md:38]: Done, will update

**Review by switzersc-usds — APPROVED** [review]
awesome, thanks for the updates!!

**Review by switzersc — APPROVED** [review]
awesome, thanks for the updates!

**Pull request merged**